ansible/roles/operator-pipeline/templates/openshift/tasks/preflight.yml

```diff
@@ -42,8 +79,6 @@ spec:
           exit 0
         fi
 
-        export KUBECONFIG=$(workspaces.kubeconfig.path)/kubeconfig
-
         preflight check operator $(params.bundle_image)
 
         echo -n preflight.log | tee $(results.log_output_file.path)
```

Comment:
It would be great if you can re-use env variables (PFLT_LOGFILE, ...) for logs, results, and artifacts.

Reply:
Fixed